content/de/cluster.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Cluster
+status: Feedback erwünscht
+category: Konzept
+tags: ["infrastruktur", "grundlegendes", ""]
+---
+
+## Was es ist
+
+Ein Cluster ist eine Gruppe von Computern oder Anwendungen, die ein gemeinsames Ziel haben.
+Im Kontext von Cloud-native Computing wird der Begriff häufig mit [Kubernetes](/kubernetes/) gleichgesetzt.
+Ein Kubernetes Cluster ist ein Satz von Services oder Workloads, die üblicherweise auf verschiedenen Maschinen in ihren eigenen Containern laufen.
+All diese [containerisierten](/containerization/) Services, die über ein Netzwerk miteinander verbunden sind, stellen einen Cluster dar.
+
+## Welches Problem es löst
+
+Software, die nur auf einem einzelnen Computer läuft, stellt einen Single Point of Failure dar. Business-kritische Systeme können offline gehen, wenn dieser Computer abstürzt oder jemand aus Versehen den Stecker zieht.
+Daher wird moderne Software grundsätzlich als verteilte Anwendung entwickelt, die als Cluster zusammen gebündelt wird.
```

Comment:
ich glaube das würde ich in den nächsten Abschnitt schieben, damit wir hier nur über das Problem sprechen

Reply:
Ich nehme an, du meinst dass "Daher wird ..." in den Lösungsteil verschoben werden sollte?

Reply:
jap

Reply:
verschoben